mmv1/products/workstations/Workstation.yaml

```
@@ -24,6 +24,15 @@ references: !ruby/object:Api::Resource::ReferenceLinks
   guides:
     "Workstations": "https://cloud.google.com/workstations/docs/"
   api: "https://cloud.google.com/workstations/docs/reference/rest/v1beta/projects.locations.workstationClusters.workstationConfigs.workstations"
+iam_policy: !ruby/object:Api::Resource::IamPolicy
+  method_name_separator: ":"
+  fetch_iam_policy_verb: :POST
+  parent_resource_attribute: workstation
```

Error is:

```
Error: Missing required argument
        
          on terraform_plugin_test.tf line 72, in resource "google_workstations_workstation_iam_policy" "foo":
          72: resource "google_workstations_workstation_iam_policy" "foo" {
        
        The argument "workstation_id" is required, but no definition was found.
```

The argument being used to reference the workstation is `workstation` but should be `workstation_id`. 

```suggestion
  parent_resource_attribute: workstation_id
```

---

Thanks, fixed it.